docs/source/courses/video-analysis.md

```diff
@@ -48,6 +48,6 @@ Useful links:
 - [keypoint-moseq](https://keypoint-moseq.readthedocs.io/en/latest/index.html)
 
 Recommended readings:
-- [Neuroscience Needs Behavior: Correcting a Reductionist Bias](https://www.sciencedirect.com/science/article/pii/S0896627316310406?via%3Dihub)
+- [Neuroscience Needs Behavior: Correcting a Reductionist Bias](https://www.cell.com/neuron/pdf/S0896-6273(16)31040-6.pdf)
```

Comment:
May be worth just using the DOI? Linking to the paper directly may cause issues if:
- The journal change the link
- Users don't have access to the paper